transfer/src/main/java/org/kinecosystem/transfer/receiver/presenter/AccountInfoError.kt

```diff
@@ -0,0 +1,15 @@
+package org.kinecosystem.transfer.receiver.presenter
+
+class AccountInfoError(val errorType: Type, val listener: IErrorActionClickListener? = null) {
+    var errorMessage: String = ""
+
+    interface IErrorActionClickListener {
+        fun onOkClicked(errorType: Type)
+    }
+
+    enum class Type {
+        None,
+        General,
+        ReLoginRequired
```

Comment:
I would call this ActionType with 2 options None, LaunchMainActivity
And add a displayErrorMessage member variable